### ISSUE TITLE
Loading Animation on FL Community Care Services

### ISSUE THREAD

**Issue by Agile6MSkinner** [department-of-veterans-affairs/va.gov-cms#19800]
## Description
On the Facility Locator, the Community Care Services list does not immediately load because it has to query PPMS. While this is happening, users may feel like they are stuck or not realize there will be service options if it takes a while to load.

Instead of immediately showing the services field We will add a loading animation as shown in [these designs](https://www.figma.com/design/GvNVeG9GAXZaBpny7vNxZP/Facility-Locator?node-id=974-3740&t=WzATuzwZKucEVQHH-4)

Existing DS loading indicator: https://design.va.gov/components/loading-indicator

### User story

**AS A** user searching for services at a community care provider
**I WANT** a loading animation to show while data is being retrieved
**SO THAT** I know that there will soon be an option for me to select a service type

### Engineering notes / background
Will use redux from #19549

## Acceptance criteria
- [x] When user selects Community Providers in facility type, the service selector box does not immediately appear
- [x] In its place, a loading animation displays and persists until PPMS data is loaded
- [x] When PPMS data loads, animation is replaced by the services selector

**Comment by jilladams** [comment]
@thejordanwood got voluntold to add the DS paradigm for loading indicator into Figma to make this ticket workable. Thank you Jordan.

**Comment by thejordanwood** [comment]
@Agile6MSkinner Dave has already shown how this should look in Figma. The loading indicator in the Figma file is from the Figma design system, but to see how it looks when animated you can view it on [this page of the design system](https://design.va.gov/components/loading-indicator).

I believe this is all that's needed, but let me know if you have anymore questions!

cc: @davidmpickett 

![Image](https://github.com/user-attachments/assets/9f559c06-1241-4321-aec5-5f1d5f5307fc)

**Comment by Agile6MSkinner** [comment]
@thejordanwood Nope, that's all! I did actually happen to see it elsewhere on va.gov today as well. Looks great

**Comment by jilladams** [comment]
This work is tested, and merged to an integration branch. We will track next steps for it merging to main under #20306. Closing.
